templates/credentials/index.html

```diff
@@ -22,8 +21,10 @@
                     background-position: 100% 60%">
       <div class="row u-equal-height">
         <div class="col-6">
-          <h1>Canonical Ubuntu Essentials</h1>
-          <p class="p-heading--4">Skills-based access to top careers</p>
+          <h1>Canonical Credentials is now Canonical University</h1>
+          <p>
+            We're expanding our exam portfolio to cover multiple job tracks aligned with real-world roles so that you can get skills-based access to top careers
```

Comment:
for consistency
```suggestion
            We're expanding our exam portfolio to cover multiple job tracks aligned with real-world roles so that you can get skills-based access to top careers.
```